benchmark/single-source/DataBenchmarks.swift

```diff
@@ -89,11 +89,29 @@ func sampleString() -> Data {
     return Data(bytes: bytes)
 }
 
+@inline(__always)
+func getRandomBuf(_ arg: UnsafeMutableBufferPointer<UInt8>) {
+#if os(Linux)
+    fatalError("Unsupported platform")
```

Comment:
This can build if:

```
#if canImport(Glibc)
import Glibc
#endif
```

and:

```
getrandom(arg.baseAddress, arg.count, 0)
```

Reply:
Ok.

Reply:
I can't do this exactly using canImport since this benchmark compiles in swift-3 mode. But I am going to do something similar.